phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -39,11 +39,8 @@
 
     @Bean
     public ViewLibrary viewLibrary() {
-        return new ViewLibrary(
-                applicationName,
-                themeConfiguration.themeManager(),
-                phoenicisGlobalConfiguration.objectMapper()
-        );
+        return new ViewLibrary(applicationName, themeConfiguration.themeManager(),
```

Comment:
It would be great if the code formatter would keep the other way of presenting beans parameters. It is a lot more readable